test/SlowTests/Issues/RavenDB_23132.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Smuggler;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23132 : RavenTestBase
+    {
+        private readonly DatabaseItemType _operateOnTypes = DatabaseItemType.Documents |
+                                                            DatabaseItemType.Indexes |
+                                                            DatabaseItemType.TimeSeries;
+
+        public RavenDB_23132(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Indexes | RavenTestCategory.TimeSeries)]
+        public async Task DeleteTimeSeriesShouldNotThrowNotSupportedException_mapReduceIndexWithOutputToCollection()
```

Comment:
Is it worth checking on both storage engines  - Lucene and Corax?

Reply:
`SearchEngineMode = RavenSearchEngineMode.All` handles only Lucene in v5.4. Do we have a better option than manually declaring Corax and Lucene separately in `[RavenData]`?

Reply:
We don't want to test Corax on 5.4. Let's handle that after merging this change to 6.0. Then you'll need to create a dedicated PR to adjust this test on 6.0

Reply:
I didn't realized this PR is for 5.4. Corax on 5.4 isn't stable.